assisted_service_mcp/src/utils/log_analyzer/signatures/error_detection.py

```diff
@@ -193,7 +193,7 @@ def analyze(self, log_analyzer) -> Optional[SignatureResult]:
         if cluster.get("high_availability_mode") != "None":
             return None
         path = (
-            "controller_logs.tar.gz/must-gather.tar.gz/must-gather.local.*/*/cluster-scoped-resources/"
+            "controller_logs.tar.gz/must-gather.tar.gz/must-gather.local.*/quay.io-openshift-release-dev-*/cluster-scoped-resources/"
```

Comment:
In a log bundle I downloaded, I saw it was with a dash instead of a period

```suggestion
            "controller_logs.tar.gz/must-gather.tar.gz/must-gather.local.*/quay-io-openshift-release-dev-*/cluster-scoped-resources/"
```

Reply:
oops, good catch. Thanks